sympy/integrals/tests/test_prde.py

```diff
@@ -52,7 +53,8 @@ def test_prde_special_denom():
     assert prde_special_denom(Poly(5*x*t + 1, t), Poly(t**2 + 2*x**3*t, t), Poly(t**3 + 2, t), G, DE) == \
         (Poly(5*x*t + 1, t), Poly(0, t), [(Poly(t, t), Poly(t**2, t)),
         (Poly(2*t, t), Poly(t, t))], Poly(1, x))
-    DE = DifferentialExtension(extension={'D': [Poly(1, x), Poly((t**2 + 1)*2*x, t)]})
+    DE = DifferentialExtension(extension={'D': [Poly(1, x), Poly((t**2 + 1)*2*x, t)]
```

Comment:
I am guessing we should have test cases for `parametric_log_deriv` corresponding to this (i.e whatever input `parametric_log_deriv` received for this test case)? Since this problem will eventually be tried (and in most of the cases be solved successfully) by "heuristic version" and never reach the "deterministic version" (after we put `try, except` block.